tut/Strategies.ipynb

```diff
@@ -1464,6 +1464,106 @@
     "print(selector[0].select_action(7))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Simulating strategies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `Simulator` class in `fimdp.core` module allows for running a simulation using a given strategy object and its corresponding `ConsMDP`. The class a strategy object and the number of simulation steps as input and performs the simulation while storing the states and actions in the `state_history` and `action_history` attributes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import sample ConsMDP and synthesize a CounterStrategy\n",
+    "from fimdp.examples.reachability_examples import ultimate\n",
+    "from fimdp.energy_solvers import BasicES, BUCHI\n",
+    "m, T = ultimate()\n",
+    "solver = BasicES(m, cap=40, targets=T)\n",
+    "m.energy_levels = solver\n",
+    "selector = solver.get_selector(BUCHI)\n",
+    "from fimdp.core import CounterStrategy\n",
+    "strategy = CounterStrategy(m, selector, 40, 10, init_state=6)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The state history of the simulation is [6, 8, 5, 4, 5, 3]\n",
+      "The action history of the simulation is ['B', 'r', 'r', '', 't']\n"
+     ]
+    }
+   ],
+   "source": [
+    "# simulate\n",
+    "from fimdp.core import Simulator\n",
+    "sim = Simulator(strategy, 5)\n",
+    "print('The state history of the simulation is {}'.format(sim.state_history))\n",
```

Comment:
In order to be consistent with the style of the other notebooks, I will change the formated strings into f-strings.